internal/metadatastorage/mysqlstore/mysql.go

```diff
@@ -150,14 +157,30 @@ func (s *Store) Store(ctx context.Context, gitoid string, obj []byte) error {
 		}
 
 		for _, sig := range envelope.Signatures {
-			_, err = tx.Signature.Create().
+			storedSig, err := tx.Signature.Create().
 				SetKeyID(sig.KeyID).
 				SetSignature(base64.StdEncoding.EncodeToString(sig.Signature)).
 				SetDsse(dsse).
 				Save(ctx)
 			if err != nil {
 				return err
 			}
+
+			for _, timestamp := range sig.Timestamps {
+				timestampedTime, err := timeFromTimestamp(timestamp)
```

Comment:
Correct me if I am wrong, but we will only store the time if a timestamp exists.

Reply:
Correct. I’m splitting up the other time stuff currently